packages/prosess/vedtak/src/VedtakProsessIndex.tsx

```diff
@@ -24,54 +22,33 @@ import messages from '../i18n/nb_NO.json';
 const intl = createIntl(messages);
 
 interface Props {
-  beregningresultatDagytelse?: BeregningsresultatDagytelse;
-  beregningresultatEngangsstonad?: BeregningsresultatEs;
+  beregningsresultat?: BeregningsresultatDagytelse | BeregningsresultatEs;
+  originaltBeregningsresultat?: BeregningsresultatDagytelse | BeregningsresultatEs;
   tilbakekrevingvalg?: TilbakekrevingValg;
   simuleringResultat?: SimuleringResultat;
-  beregningsgrunnlag?: Beregningsgrunnlag;
-  beregningsresultatOriginalBehandling?: {
-    'beregningsresultat-engangsstonad'?: BeregningsresultatEs;
-    'beregningsresultat-foreldrepenger'?: BeregningsresultatDagytelse;
-  };
+  beregningErManueltFastsatt: boolean;
   vilkar: Vilkar[];
   previewCallback: (data: ForhandsvisData) => void;
   oppgaver?: Oppgave[];
   ferdigstillOppgave: (oppgaveId: OppgaveId) => Promise<void>;
 }
 
 export const VedtakProsessIndex = ({
-  beregningresultatDagytelse,
-  beregningresultatEngangsstonad,
+  beregningsresultat,
+  originaltBeregningsresultat,
   tilbakekrevingvalg,
   simuleringResultat,
-  beregningsgrunnlag,
+  beregningErManueltFastsatt,
   vilkar,
-  beregningsresultatOriginalBehandling,
   previewCallback,
   oppgaver,
   ferdigstillOppgave,
 }: Props) => {
-  const { behandling, fagsak } = usePanelDataContext();
-
-  const { aksjonspunkt } = behandling;
-
-  const beregningErManueltFastsatt = skalSkriveFritekstGrunnetFastsettingAvBeregning(aksjonspunkt, beregningsgrunnlag);
```

Comment:
Ok, ser no at du flytta denne funksjonaliteten frå dette panelet og opp eit hakk (eg hadde jo ein kommentar på at du skulle flytta det ned hit ein annan stad :). Eg syns vel eigentleg det passar bedre å legga dette her. Det er logikk som kun blir brukt internt i panelet, og ikkje blir brukt til noko i initPanelet (som har ansvar for om panelet skal visast, status i meny osv)